tests/org.jboss.tools.cdk.ui.bot.test/src/org/jboss/tools/cdk/ui/bot/test/server/editor/CDK3ServerEditorTest.java

```diff
@@ -90,7 +118,85 @@ private void cleanUp() {
 			serversView = null;
 		}
 	}
+	
+	private void checkEditorStateAfterSave(String location, boolean canSave) {
+		((CDK3ServerEditor) editor).getMinishiftBinaryLabel().setText(location);	
+		AbstractWait.sleep(TimePeriod.getCustom(1));
```

Comment:
I can think of 'new WaitUntil(new JobIsRunning(), TimePeriod.SHORT, false);'
But that seems to me pretty nasty. The most clean solution would be to implement new wait condition that would be checking existence of error field(maybe other) decorator, but I do not know how to reach that decorator programmatically. Nor via eclipse api or reddeer.
WDYT?